src/main/java/frc/robot/Controls.java

```diff
@@ -90,6 +88,10 @@ private void configureDrivebaseBindings() {
       // Stop running this method
       return;
     }
+
+    double getLeftX = Math.pow(MathUtil.applyDeadband(driverController.getLeftX(), 0.1), 3);
+    double getLeftY = Math.pow(MathUtil.applyDeadband(driverController.getLeftY(), 0.1), 3);
+    double getRightX = Math.pow(MathUtil.applyDeadband(driverController.getRightX(), 0.1), 3);
```

Comment:
Make sure to test these with Strummer before checking in. Searching https://github.com/search?q=driverController.getLeftX+AND+Math.pow&type=code I see some teams doing a cube here, some doing a square, and from https://github.com/search?q=driverController.getLeftX&type=code some just use the value as-is.
And [the Technotoes only cube the rotation value](https://github.com/technototes/IntoTheDeep2024/blob/6ec6b0732f9900c53915d7b13738b49e098a1901/Hoops/src/main/java/org/firstinspires/ftc/hoops/commands/JoystickDriveCommand.java#L52), and not the translation inputs.